### PR TITLE
add a few solultions and editorials

### PR DESCRIPTION
**Pull request by Amaryllidaceae**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [X] I have tested my code.
- [X] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [X] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

Add editorial and python solution to usaco: Angry Cows
Add 1 editorial and 1 python solution to cf: Cellular Network
Add editorial and python solution to usaco: Social Distancing
Add python solution to cses: Array Division
Add python solution to cf: The Meeting Place Cannot be Changed
Add python solution and clean up usaco: Milk Pumping

I had the python docstrings already from months ago so I decided to keep them in

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
hey man
could you split these into multiple prs next time
it's a pain for reviewers

File: solutions/cf-702C.mdx [hunk 1]
```diff
@@ -189,9 +198,49 @@ int main() {
...
+<PySection>
+
+```py
+n, m = map(int, input().split())
+
+# We remove all the duplicate cities and towers
+cities = sorted(list(set(map(int, input().split()))))
+towers = sorted(list(set(map(int, input().split()))))
+
+'''
```
Comment (SansPapyrus683) [inline solutions/cf-702C.mdx:211]: double quotes

File: solutions/cf-782b.mdx [hunk 2]
```diff
@@ -101,4 +101,53 @@ int main() {
...
+def all_friends_converge(seconds: int) -> bool:
+	"""
+	Checks whether all friends can converge on one point in the specified time interval.
+
+	Args:
+		seconds (int): Amount of seconds given for friends to converge.
+
+	Returns:
+		bool: If the friends can converge to a single point.
+	"""
```
Comment (SansPapyrus683) [inline solutions/cf-782b.mdx:121]:
```suggestion
	"""
	Checks whether all friends can converge on one point in the specified time interval.
	:param seconds: Amount of seconds given for friends to converge.
	:returns: If the friends can converge to a single point.
	"""
```
doc comment syntax from [here](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats)

File: solutions/cf-782b.mdx [hunk 3]
```diff
@@ -101,4 +101,53 @@ int main() {
...
+			return False
+		if lower_bound > overlap_lower:
+			overlap_lower = lower_bound
+		if upper_bound < overlap_upper:
+			overlap_upper = upper_bound
+	return True
+
+
+left, right = 0, 10 ** 9
+diff = 10 ** (-6)
```
Comment (SansPapyrus683) [inline solutions/cf-782b.mdx:137]: are the parens needed?

File: solutions/cses-1085.mdx [hunk 4]
```diff
@@ -90,3 +95,58 @@ int main(){
...
+	"""
+	Checks if it is possible to divide nums into k subarrays with each
+	each subarray having a maximum sum of max_sum
+
+	Args:
+		max_sum (int): The maximum sum of each subarray
+
+	Returns:
+		bool: If it is possible to divide nums with the above conditions.
+	"""
```
Comment (SansPapyrus683) [inline solutions/cses-1085.mdx:117]: same w/ above

File: solutions/usaco-594.mdx [hunk 5]
```diff
@@ -59,4 +74,62 @@ int main () {
...
+```py
+with open('angry.in', 'r') as infile:
+	n, k = map(int, infile.readline().split())
+	bales = [int(infile.readline()) for _ in range(n)]
+	bales.sort()
+
+
+def valid_blast_radius(blast_radius: int) -> bool:
+	"""
+	Tests if all bales can be blown up with the specified blast radius.
```
Comment (SansPapyrus683) [inline solutions/usaco-594.mdx:89]: here too

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
yeah this code is really good

File: solutions/cf-782b.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/cf-782b.mdx:121]: oh wait this seems like the google syntax
uh most of the docstrings here follow the reST syntax

**Comment by Amaryllidaceae** [comment]
To clarify, each solution with editorial should be it's own pull request?

File: solutions/usaco-1038.mdx [hunk 6]
```diff
@@ -73,6 +81,44 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+with open('socdist.in', 'r') as r:
+	n, m = map(int, r.readline().split())
+	intervals = [tuple(map(int, r.readline().split())) for _ in range(m)]
+	intervals.sort(key=lambda i: i[0])
```
Comment (SansPapyrus683) [inline solutions/usaco-1038.mdx:90]: don't think custom comparator here is necessary

**Comment by SansPapyrus683** [comment]
> To clarify, each solution with editorial should be it's own pull request?

ye

**Review by SansPapyrus683 — APPROVED** [review]
all the code still works right

**Comment by Amaryllidaceae** [comment]
yeup, do you want screenshots again?

**Comment by SansPapyrus683** [comment]
nah just wanted to make sure
@ryanchou-dev

**Review by ryanchou-dev — APPROVED** [review]
alr

**Pull request merged**